packages/pretty-format/src/__tests__/html_element.test.js

```diff
@@ -16,6 +17,7 @@ const toPrettyPrintTo = require('./expect_util').getPrettyPrint([
   HTMLElementPlugin,
 ]);
 
+const expect: any = global.expect;
```

Comment:
does any one have any ideas how to flowtype custom matchers?

i don't know if there's a way to do something like this:
```js
const expect: Expect & MyCustomMatcherType = global.expect;
```

or maybe it's better to detach it from `expect` to have a consistent typing like
```js
const expectToBeSomething = expect.extend(matcherFunction);

test('custom test', () => {
  expectToBeSomething(received, actual);
});
```